inc/functions.php

```diff
@@ -308,7 +308,7 @@ function register_editor_assets( WP_Block_Type $block ): void {
 /**
  * Get highlight theme name.
  *
- * @return string Theme name.
+ * @return string|null Theme name or null if disabled.
```

Comment:
Ah, this won't work because of the PHP `string` return type. So it will need to rather be:

```suggestion
 * @return string Theme name or empty string if disabled.
```